app/imports/api/sprints/methods.js

```diff
@@ -1,15 +1,35 @@
+import { Meteor } from 'meteor/meteor';
 import { ValidatedMethod } from 'meteor/mdg:validated-method';
+import { isProjectModerator } from '/imports/api/projects/helpers.js';
+import { Projects } from '/imports/api/projects';
 
 import { Sprints } from './Sprints.js';
-import {
-    SprintSchema,
-} from './schema.js';
+import { SprintSchema } from './schema.js';
 
 export const addSprint = new ValidatedMethod({
     name: 'sprints.add',
     validate: SprintSchema.validator({ clean: true }),
     run({ name, projectId }) {
-        // @todo add check if user is moderator
-        return Sprints.insert({ name, projectId });
+        const projects = Projects.findOne(projectId);
+        const userId = Meteor.userId();
+        if (isProjectModerator({ projects, userId })) {
```

Comment:
Change `isProjectModerator({ projects, userId })` to ` isProjectModerator(projects, userId)` (see function `isProjectModerator`)